docs/articles/documentation/test-api/actions/select-text.md

```diff
@@ -0,0 +1,115 @@
+`selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the textarea whose text will be selected. See [Selecting Target Elements](index.md#selecting-target-elements).
+`startLine`| Numeric                                           | The line number at which selection will start.
+`startPos` | Numeric                                           | The start position of selection within the line defined by the `startLine`.
+`endLine`  | Numeric                                           | The line number at which selection will end.
+`endPos`   | Numeric                                           | The end position of selection within the line defined by `endLine`.
+
+The following example shows how to select text within a \<textarea\> element.
+
+```js
+import { expect } from 'chai';
+import { Selector } from 'testcafe';
+
+const getComment = Selector(id => document.querySelector('#comment'));
+
+test('Select text within textarea', async t => {
+    await t.selectTextAreaContent(getComment, 1, 5, 3, 10);
+
+    const comment = await getComment();
+
+    expect(comment.attributes['selectionStart']).to.equal(5);
+    expect(comment.attributes['selectionEnd']).to.equal(48);
+});
+```
+
+> `startPos`, `endPos`, `startLine` and `endLine` cannot have negative values.
+>
+> If the position defined by `startLine` and `startPos` is greater than the one defined
+> by `endLine` and `endPos`, the action will perform backward selection.
+
+## Perform Selection within Editable Content
+
+```text
+selectEditableContent( startSelector, endSelector )
+```
+
+Parameter       | Type                                              | Description
+--------------- | ------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`startSelector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies a webpage element from which the selection starts. The start position of selection is the first character of the element's text. See [Selecting Target Elements](index.md#selecting-target-elements).  
+`endSelector`   | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies a webpage element at which the selection ends. The end position of selection is the last character of the element's text. See [Selecting Target Elements](index.md#selecting-target-elements).
+
+This function works for selection inside HTML elements that have the `contentEditable` attribute enabled.
+
+> Important! According to Web standards, start and end elements must have a common ancestor with the `contentEditable` attribute set to `true` -
+> a [root container](https://www.w3.org/TR/DOM-Level-2-Traversal-Range/ranges.html#td-root-container).
```

Comment:
Don't reference W3C spec, it's outdated. Use WHATWG living standards: https://dom.spec.whatwg.org/#concept-range-bp